src/features/list-view/examples/ListViewExamples.js

```diff
@@ -7,21 +7,21 @@ import ListView from '../ListView';
 
 const generateRandomString = value => String.fromCharCode((value % 65) + 65);
 
-const getGridHeader = columnIndex => generateRandomString(columnIndex);
+const getGridHeader = columnIndex => `Header ${generateRandomString(columnIndex)}`;
 
-const getGridCell = ({ columnIndex, rowIndex }) => generateRandomString(columnIndex + rowIndex);
+const getGridCell = ({ columnIndex, rowIndex }) => `Row ${rowIndex}, Column ${columnIndex}`;
 
 const ListViewExamples = () => {
     return (
-        <div style={{ height: '1000px' }}>
+        <div style={{ height: '300px' }}>
             <AutoSizer>
                 {({ height, width }) => (
                     <ListView
                         columnCount={1000}
                         height={height}
                         getGridCell={getGridCell}
                         getGridHeader={getGridHeader}
-                        rowCount={35}
+                        rowCount={1000}
```

Comment:
Could this and columnCount also be moved to be a constant?

Reply:
Same.